src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx

```diff
@@ -65,28 +47,48 @@ export function BackUpSecretKeyLayout(props: BackUpSecretKeyLayoutProps): JSX.El
           </Stack>
         </Flex>
         <Flex
-          className={cx(
-            { [fullPageOnboardingContent]: isFullPage },
-            { [popupPageContent]: isPopup }
-          )}
+          alignItems={['start', 'center']}
           flexGrow={1}
           justifyContent="center"
+          mt={['base', 'unset']}
         >
-          <Box
-            border="1px solid"
-            borderColor={color('border')}
-            borderRadius="16px"
-            height="484px"
-            width="432px"
-          >
-            <Stack alignItems="center" pt="80px" spacing="loose">
-              <KeyIllustration />
+          <Box border="1px solid" borderColor={color('border')} borderRadius="16px" width="446px">
+            <Stack
+              alignItems="center"
+              p={['base-loose', 'extra-loose']}
+              ml={['tight', 'extra-tight']}
+              spacing="loose"
+            >
+              <Box width={['87px', '101px']}>
+                <img src={YourSecretKey} />
+              </Box>
               <Title fontSize="20px">Your Secret Key</Title>
-              <Text px="loose" textAlign="center">
-                {secretKey}
-              </Text>
-              <Link fontSize="14px" onClick={!hasCopied ? onCopyToClipboard : undefined}>
-                {!hasCopied ? 'Copy to clipboard' : 'Copied!'}
+              <Stack justifyContent="center" isInline wrap="wrap">
+                {secretKey?.split(' ').map(word => {
+                  return (
+                    <Box
+                      border="1px solid"
+                      borderColor={color('border')}
+                      borderRadius="8px"
+                      mb="8px !important"
+                      p="tight"
+                    >
+                      <Text>{word}</Text>
+                    </Box>
+                  );
+                })}
+              </Stack>
```

Comment:
Let's make a new component for each word, and perform splitting logic outside the JSX.

Reply:
Yep, good call, will do.